### ISSUE TITLE
Better avendesora error message?

### ISSUE THREAD

**Issue by kalekundert** [KenKundert/emborg#4]
It took me a while to figure out that the following error was refering to a problem with my avendesora config, not with emborg or borg itself.

```
$ emborg init
emborg warning: rsync: url attribute found, should probably be urls.
emborg error: Connection closed by remote host. Is borg working on the server?
```

**Comment by kalekundert** [comment]
Update: The "connection closed" is unrelated to the "url" thing.  Looking into that now.

**Comment by kalekundert** [comment]
And the "url" thing was me being stupid (I was defining "repository" in `settings` and not realizing that it was also defined in `home`).

**Comment by KenKundert** [comment]
`emborg warning: rsync: url attribute found, should probably be urls.`

That message comes from Avendesora. It says that in your rsync account an account attribute 'url,' which is commonly mistakenly used when 'urls' is intended.  I don't know why you would be using the urls attribute with an rsync account. Did you really want to use url? If so I can remove that warning message.

`emborg error: Connection closed by remote host. Is borg working on the server?`

I believe the body of that message is generated by borg. Perhaps that is the error you fixed with the change to the 'repository' setting? It is okay to have settings defined in both settings and in home. The one in home should dominate.

**Comment by kalekundert** [comment]
Yeah, the confusing thing is that it says "emborg warning", when really it's an avendesora warning.  No need to remove the warning, it's a good warning, it just confused me in this context.

I put a URL in my rsync.net account because when you create an account, they assign you a fairly non-intuitive URL, and I just wanted to store that somewhere.
